cmd/webhook-manager/app/server.go

```diff
@@ -101,7 +102,7 @@ func Run(config *options.Config) error {
 	}
 	go func() {
 		err = server.ListenAndServeTLS("", "")
-		if err != nil && err != http.ErrServerClosed {
+		if err != nil && !errors.Is(err, http.ErrServerClosed) {
```

Comment:
IMO, this change does not seem to change any logic. Can you explain the benefits of such a change?

Reply:
you r right, i have remove this change, we dont need this change now